sdk/batch/batch/test/batchClient.spec.ts

```diff
@@ -393,19 +395,28 @@ describe("Batch Service", () => {
 
     it("should get pool node counts successfully", async () => {
       let result: AccountListPoolNodeCountsResponse;
-      while (true) {
+      while (true) 
+      {
         result = await client.account.listPoolNodeCounts();
         if (result.length > 0 && result[0].dedicated!.idle > 0) {
           break;
         } else {
           await wait(POLLING_INTERVAL);
         }
       }
+
       assert.lengthOf(result, 2);
-      assert.equal(result[0].poolId, ENDPOINT_POOL);
-      assert.equal(result[0].dedicated!.idle, 1);
-      assert.equal(result[0].lowPriority!.total, 0);
       assert.equal(result._response.status, 200);
+
+      const endpointPoolObj = result.filter(pool => pool.poolId == ENDPOINT_POOL);
+      if (endpointPoolObj.length == 0)
+      {
+        assert.fail(`Pool with Pool Id ${ENDPOINT_POOL} not found`);
+      }
```

Comment:
NIT: this could be more concise:
```suggestion
      assert.isAbove(endpointPoolObj.length, 0, `Pool with Pool Id ${ENDPOINT_POOL} not found`);
```

Reply:
Yes that is a cleaner approach, will make the change!